src/js/view.js

```diff
@@ -14,38 +14,77 @@ class View {
     this.$lottiesPanel = $.qs(".lottie-panel");
     this.$lottieControlPanel = $.qs(".lottie-control-panel");
     this.$lottieListContainer = $.qs(".lottie-list");
+    this.$numberCheckForm = $.qs(".number-check-form");
+    this.$winningNumberInputs = $.qs(".winning-number-inputs");
+    this.$bonnusWinningNumberInput = $.qs(".bonus-number");
+
+    // modal component
+    this.$modal = $.qs(".modal");
+    this.$modalCloseBtn = $.qs(".modal-close");
+    this.$showResultBtn = $.qs(".open-result-modal-button");
+    this.$lottoResetBtn = $.qs(".lotto-reset-btn");
 
     this.initObservable();
     this.initEvent();
   }
 
   initObservable() {
     observable.subscribe(notifyTypes.BUY_LOTTIES, this.render.bind(this));
-    observable.subscribe(
-      notifyTypes.TOGGLE_SHOW_LOTTIES_NUMBERS,
-      this.renderLottieList.bind(this)
-    );
+    observable.subscribe(notifyTypes.TOGGLE_SHOW_LOTTIES_NUMBERS, this.renderLottieList.bind(this));
+    observable.subscribe(notifyTypes.SHOW_LOTTIE_WINNING_INFO, this.renderResultModal.bind(this));
+    observable.subscribe(notifyTypes.RESET_LOTTIE_GAME, this.resetRender.bind(this));
   }
 
   initEvent() {
-    this.$buyBtn.addEventListener("click", () => {
-      this.lottoController.handleBuyLottiesBtnClick(this.$priceInput.value);
+    this.$buyBtn.addEventListener("click", this.onLottieBuy);
+    this.$showResultBtn.addEventListener("click", this.onModalOpen);
+    this.$modalCloseBtn.addEventListener("click", this.onModalClose);
+    this.$winningNumberInputs.addEventListener("input", ({ target }) => {
+      if (target.tagName === "INPUT") {
+        this.onWinningLottoDigitsInput(target);
+      }
     });
+    this.$bonnusWinningNumberInput.addEventListener("input", ({ target }) => {
+      this.onWinningLottoDigitsInput(target);
+    });
+    this.$lottoResetBtn.addEventListener("click", this.onLottoGameReset);
   }
 
+  onLottieBuy = () => {
+    this.lottoController.handleBuyLottiesBtnClick(this.$priceInput.value);
+  };
+
+  onModalOpen = () => {
+    if (!this.lottoController.canShowModal()) {
+      alert("당첨 번호이 형식이 옳바르지 않습니다");
+      return;
+    }
+    this.$modal.addClass("open");
+    this.lottoController.handleModalOpen();
+  };
```

Comment:
onModalOpen 메서드는 모달을 보여주는 한 가지 일만 수행하도록 하면 어떨까요?
```suggestion
  // 더 좋은 이름이 있을 것 같네요
  onResultShow = () => {
      if (!this.lottoController.canShowModal()) {
        alert("당첨 번호이 형식이 옳바르지 않습니다");
        return;
      }
      onModalOpen()
  }
  
  onModalOpen = () => {
    this.$modal.addClass("open");
    this.lottoController.handleModalOpen();
  };
```